### PR TITLE
chore: some changes to the migration docs

### PR DESCRIPTION
**Pull request by edgesoft**
- immediate
- compute
- paths in state

### CODE REVIEW

**Comment by edgesoft** [comment]
@Guria Can you take a look?

**Comment by edgesoft** [comment]
https://github.com/cerebral/cerebral/pull/362

File: content/migration/01_start_migration.en.md [hunk 1]
```diff
@@ -163,7 +165,46 @@ myAction.async = true
...
+
+You could do this in an action.
+
+```js
+function myAction({state}) {
+  state.set('app.nodes.some.state.more', {updated: true})
+}
+```
+
+With 2.x this is no longer possible. It's easy to mistype paths and therefore this option is no longer available. Your paths need to be in place before setting values to them.
```
Comment (henri-hulski) [inline content/migration/01_start_migration.en.md:188]: Are you sure? 
I think this discussion was about paths in function tree not state tree.

**Review by henri-hulski — DISMISSED** [review]

File: content/migration/01_start_migration.en.md (hunk 1 above)
Comment (Guria) [reply content/migration/01_start_migration.en.md:188]: @henri-hulski yes, he is right, it was discussed also as well

Comment (henri-hulski) [reply content/migration/01_start_migration.en.md:188]: Ooh sorry. 😃

File: content/migration/01_start_migration.en.md [hunk 2]
```diff
@@ -100,6 +100,8 @@ const controller = Controller({
 })
 ```
 
+One note when using signals is that the **immediate** option is gone. This keeps the api cleaner.
```
Comment (Guria) [inline content/migration/01_start_migration.en.md:103]: not just gone, but there is no need in it.

**Review by Guria — APPROVED** [review]

**Pull request merged**